src/StatPlots.jl

```diff
@@ -7,6 +7,9 @@ import Plots: _cycle
 using StatsBase
 using Distributions
 using DataFrames
+using IterableTables
```

Comment:
I would make this ``import IterableTables``. You really just want the package to be loaded, but you don't need anything from it directly.